src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerIndividualValidatorTest.java

```diff
@@ -348,11 +350,16 @@ void testNoErrorWhenCeasedDateIsSameAsStartDate() {
         assertFalse(errors.hasErrors());
     }
 
+    //Added for UAR-1595 can remove test when FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC is removed
     @Test
-    void testErrorReportedWhenNoNatureOfControlValuesAreAllNull() {
+    void testErrorReportedWhenNoNatureOfControlValuesAreAllNullWhenNewNocsFlagIsFalse() {
```

Comment:
can the test name be changed - i don't understand this one (or the previous one before the changes)
I think it just needs the No removing
does testErrorReportedWhenNatureOfControlValuesAreAllNullWhenNewNocsFlagIsFalse sound better?

Reply:
i think the 'No" removal from the name applies to a few in this class.

Reply:
Changed names on 4 tests.